src/vm/arm64/stubs.cpp

```diff
@@ -1606,6 +1606,8 @@ VOID StubLinkerCPU::EmitShuffleThunk(ShuffleEntry *pShuffleEntryArray)
     // field and save it in x9. Tailcall to the target method after re-arranging the arguments
     // ldr x9, [x0, #offsetof(DelegateObject, _methodPtrAux)]
     EmitLoadStoreRegImm(eLOAD, IntReg(9), IntReg(0), DelegateObject::GetOffsetOfMethodPtrAux());
+    //add x11, x0, DelegateObject::GetOffsetOfMethodPtrAux() - load the indirection cell into r11
```

Comment:
change r11 to x11. Add comment that VSD resolveworker expects indirection cell to be in x11